### PR TITLE
Improvement - new color dark theme and Palette doc page

### PR DESCRIPTION
**Pull request by Cuervino**
**Component**:
Color Palette - Theme.js & a new Palette documentation display

**Description**:
- A new Theme "DarkRebrand" in Theme.js 
- A new Palette display as documentation

**Design**:

Code | Name
-- | --
#E84855 | statusCritical
#F8F32B | statusWarning
#0AADA6 | statusHealthy
#037AFF | selectedActive
#1A3C75 | highlight
#8E8EAC | infoPrimary
#333366 | infoSecondary
#2F4185 | buttonPrimary
#595A78 | buttonSecondary
#3D0808 | buttonDelete
#121219 | backgroundZ1
#323245 | backgroundZ2
#232331 | backgroundZ3
#171721 | backgroundZ4
#FFFFFF | textPrimary
#EAEAEA | textSecondary
#B5B5B5 | textTertiary
#71AEFF | textLink
#313131 | border

***Screenshot***
image

**Breaking Changes**:
Yes
- [] Breaking Changes

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

File: src/lib/style/theme.js [hunk 1]
```diff
@@ -34,6 +34,32 @@ export const grayLightest = '#E9EBED';
 
 // Export the default light & dark theme
 export const defaultTheme = {
+  dark: {
+    danger: '#E84855',
+    warning: '#F8F32B',
+    healthyLight: '#0AADA6',
```
Comment (ChengYanJin) [inline src/lib/style/theme.js]: I am just thinking since we will degrade the `healthy` color, shall we rename `healthLight` to `healthy` here?

**Review by ChengYanJin — COMMENTED** [review]
Hi @Cuervino 
Thanks for updating the code as well !!! 💯
Just a reminder we may NOT want to merge this PR until we fully test the theme in all the apps. We will let you know! thanks.

File: src/lib/style/theme.js (hunk 1 above)
Comment (Cuervino) [reply src/lib/style/theme.js]: Yes, you're right.

**Comment by Cuervino** [comment]
@ChengYanJin @JBWatenbergScality I created a third theme. Comment/review welcomed :-)

File: src/lib/style/theme.js [hunk 2]
```diff
@@ -82,6 +82,28 @@ export const defaultTheme = {
...
+    border: '#313131',
+    buttonPrimary: '#2F4185',
+    buttonSecondary: '#595A78',
+    buttonDelete: '#3D0808',
+    infoPrimary: '#8E8EAC',
+    infoSecondary: '#333366',
+    backgroundZ1: '#121219',
+    backgroundZ2: '#323245',
+    backgroundZ3: '#232331',
+    backgroundZ4: '#171721',
```
Comment (JBWatenbergScality) [inline src/lib/style/theme.js]: I'm not convinced by those labels. IMO we should reflect a bit more their use case in their labels or explicit their hierarchy via a Level label (eg `backgroungLevel1`) .

File: src/lib/style/theme.js [hunk 3]
```diff
@@ -82,6 +82,28 @@ export const defaultTheme = {
     border: '#313131',
     info: '#434343',
   },
+  darkRebrand: {
+    statusHealthy: '#0AADA6',
+    statusWarning: '#F8F32B',
+    statusCritical: '#E84855',
+    selectedActive: '#037AFF',
+    highlight: '#1A3C75',
```
Comment (JBWatenbergScality) [inline src/lib/style/theme.js:90]: Is it applicable to all the contexts ? (Text highlight, border for focused cards, focus on textual link, etc..)

Comment (Cuervino) [reply src/lib/style/theme.js:90]: Most of the time, it's a combination of `selectedActive` and `highlight`.

File: src/lib/style/theme.js (hunk 2 above)
Comment (Cuervino) [reply src/lib/style/theme.js]: agree, let's take `backgroundLevel1`, etc.

**Review by JBWatenbergScality — APPROVED** [review]

**Review by ChengYanJin — APPROVED** [review]

**Pull request merged**